libsrc/Makefile

```diff
@@ -91,7 +93,7 @@ define INSTALL_recipe
 
 $(if $(prefix),,$(error variable `prefix' must be set))
 $(INSTALL) -d $(DESTDIR)$(datadir)/$(dir)
```

Comment:
Moreover some lines earlier $datadir is set as `datadir = $(prefix)/share/cc65`, which means that if somebody supplies $DESTDIR the output will go lalaland direction as $prefix can't be empty at the same time